packages/eas-cli/src/commands/submit.ts

```diff
@@ -211,3 +213,22 @@ export default class BuildSubmit extends Command {
     }
   }
 }
+
+async function promptForPlatformAsync(): Promise<SubmissionPlatform> {
+  const { platform } = await promptAsync({
+    type: 'select',
+    message: 'For which platform you want to submit a build?',
```

Comment:
```suggestion
    message: 'Choose a platform to submit to',
```
Or maybe even `Submit to platform`